### PR TITLE
Add config option for 'run_ansible_lint'

### PR DESCRIPTION
**Pull request by alikins**

### CODE REVIEW

File: galaxy_importer/collection.py [hunk 1]
```diff
@@ -220,6 +222,19 @@ def _build_contents_blob(self):
 
     def _build_docs_blob(self):
         """Build importer result docs_blob from collection documentation."""
+
+        # return an empty DocsBlob if run_ansible_doc=False
+        # if not self.cfg.run_ansible_doc:
```
Comment (awcrosby) [inline galaxy_importer/collection.py:227]: not sure if this comment line is meant to stick around

**Comment by chouseknecht** [comment]
@alikins 

Please add change log snippet and an update to the README example config. Thanks!

File: galaxy_importer/collection.py (hunk 1 above)
Comment (cutwater) [reply galaxy_importer/collection.py:227]: +1, please don't leave commented unused blocks of code.

**Comment by awcrosby** [comment]
Looks like it still needs the README example config updated with the 2 config options, otherwise looks good

**Pull request merged**